compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/focus/FocusOwnerImpl.kt

```diff
@@ -264,7 +265,8 @@ internal class FocusOwnerImpl(
 
         val activeFocusTarget = rootFocusNode.findActiveFocusNode()
         val focusedKeyInputNode = activeFocusTarget?.lastLocalKeyInputNode()
-            ?: activeFocusTarget?.nearestAncestorIncludingSelf(Nodes.KeyInput)?.node
+                ?: activeFocusTarget?.nearestAncestorIncludingSelf(Nodes.KeyInput)?.node
+                ?: rootFocusNode.nearestAncestor(Nodes.KeyInput)?.node
```

Comment:
Does it work with our root focus? Do we have some test for it?

Reply:
It should be covered by [ComposeFocusTest](https://teamcity.jetbrains.com/buildConfiguration/JetBrainsPublicProjects_Compose_DesktopTests/4735781?buildTab=tests&name=ComposeFocusTest)